graylog2-server/src/main/java/org/graylog2/alarmcallbacks/EmailAlarmCallback.java

```diff
@@ -85,7 +74,7 @@ public void call(Stream stream, AlertCondition.CheckResult result) {
                         .addType(Notification.Type.EMAIL_TRANSPORT_CONFIGURATION_INVALID)
                         .addSeverity(Notification.Severity.NORMAL)
                         .addDetail("stream_id", stream.getId())
-                        .addDetail("exception", Throwables.getStackTraceAsString(e));
+                        .addDetail("exception", e.getMessage());
```

Comment:
Generally :+1:, but the actual helpful message might be buried in the root cause (i. e. `e.getCause().getMessage()`).

[`Throwables#getRootCause(java.lang.Throwable)`](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/Throwables.html#getRootCause(java.lang.Throwable)) might be helpful for this case.